tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/PublicClientApplicationTests.cs

```diff
@@ -1262,6 +1262,38 @@ public void EnsurePublicApiSurfaceExistsOnInterface()
             CheckBuilderCommonMethods(byRefreshTokenBuilder);
         }
 
+        [TestMethod]
+        public void CheckUserProvidedCorrlationIDTest()
+        {
+            using (var harness = CreateTestHarness())
+            {
+                harness.HttpManager.AddInstanceDiscoveryMockHandler();
+                var correlationId = new Guid();
+                PublicClientApplication app = PublicClientApplicationBuilder.Create(MsalTestConstants.ClientId)

```

Comment:
nit:  please wrap these to the next line tabbed in 4 spaces to reduce width for code reviews:

PublicClientApplication app = PUblicClientApplicationBuilder
    .Create(foo)
    .WithAuthority(bar)
    .etc();